.gitattributes

```diff
@@ -118,3 +118,4 @@
 *.zip filter=lfs diff=lfs merge=lfs -text
 *.blend filter=lfs diff=lfs merge=lfs -text
 *.xcf filter=lfs diff=lfs merge=lfs -text
+*.tga filter=lfs diff=lfs merge=lfs -text
```

Comment:
.tga is already on line 107 - also this file should be in alphabetical order to make it easier to find extensions.

Reply:
Removed.